seL4-platforms/platforms.yml

```diff
@@ -208,7 +208,7 @@ platforms:
     platform: rpi3
     req: [rpi3]
     image_platform: bcm2837
-    march: armv8a
+    march: armv8a # ARMv8 platform currently used in AARCH32 mode only.
```

Comment:
This probably means the build scripts will override the setting to `armv7a`. I assume the file here lists `armv8a` because the intention originally was to use it in 64bit mode, but that then didn't happen.

I'd vote for fixing this here, so that build info and actual build are in sync. Nothing particularly bad happens when they are not, but it's confusing and logs etc will be harder to find.

Reply:
(we should check first what the build actually does, one sec)

Reply:
Nope, I'm wrong, the cmake config also sets `armv8a`. So good to leave as is here.